r/tests/testthat/test-metadata.R

```diff
@@ -383,6 +383,7 @@ test_that("grouped_df metadata is recorded (efficiently)", {
   grouped_tab <- Table$create(grouped)
   expect_r6_class(grouped_tab, "Table")
   expect_equal(grouped_tab$r_metadata$attributes$.group_vars, "a")
+  expect_equal(grouped_tab$metadata$r$attributes$.group_vars, "a")
```

Comment:
L385 and L386 are the same thing, we don't need both tests (pick one, either is fine)

Reply:
I interpreted this test being left in place despite the change in metadata structure (`r_metadata` to `metadata$r`) as being for compatibility, but is it safe to remove?

Reply:
Yeah, we test elsewhere that `r_metadata == metadata$r`, don't need to test it again here.